samples/tfm_integration/tfm_secure_partition/README.rst

```diff
@@ -1,7 +1,7 @@
-.. _tfm_secure_partition:
+.. zephyr:code-sample:: tfm_secure_partition
+   :name: TF-M Secure Partition
```

Comment:
Maybe no need for capitalization in every word here? (not sure how TF-M defines it)

Reply:
I just went with the way it was capitalized in the overview section a few lines below :)
I think it's fair to say capitalized version should be ok https://tf-m-user-guide.trustedfirmware.org/integration_guide/services/tfm_secure_partition_addition.html